MAINTAINERS.yml

```diff
@@ -2013,6 +2446,8 @@ Native POSIX/Sim and POSIX arch:
     - "area: native port"
   description: >-
     POSIX architecture and SOC, native_posix & native_sim boards, and related drivers
+  tests:
+    - boards.native_posix
```

Comment:
could you name this board.native_sim instead? (given that we are deprecating the native_posix board..)

Reply:
yep, had this change for a while before the change to native_sim, change now.